src/library_noderawfs.js

```diff
@@ -95,10 +95,10 @@ addToLibrary({
       // -1 here for atime or mtime means UTIME_OMIT was passed.  Since node
```

Comment:
I guess this comment needs updating too.

Any reason to switch from `-1` to `undefined` ?  (That later take a little more code size).

Reply:
Well it's useful that undefined is falsey. If we're after minimizing code size, in this block I removed the two conditionals which saved ~22 characters and switched -1 to undefined, which added 14 characters so it was a net improvement. We could save 10 more characters if you want by testing `if ((atime ?? mtime) === undefined)`

Reply:
What about using `null` that's only 4 characters.

Reply:
Oh nevermind `if ((atime ?? mtime) === undefined)` is the conditional with `&&`, this is `||`...